projects/managed-hubs/support.md

```diff
@@ -1,118 +1,214 @@
+- Add new users to a hub, including administrative users.
+- Surface major issues or requests to the Community Representative(s).
 
 ## Communication channels
 
-We use two Slack channels to discuss support-related questions on the 2i2c hub:
+### External communication
 
-- The [#support-freshdesk](https://2i2c.slack.com/archives/C028WU9PFBN) is for discussing support tickets on Freshdesk.
-  Open a Slack thread for each support ticket, to have conversation around it amongst the team.
-  This channel is private because there’s an assumption of privacy when we discuss community-specific support matters.
-- The [#hub-support](https://2i2c.slack.com/archives/C01DB2JRP8W) channel is for discussing more general support-related issues relating to operating the 2i2c Hubs, and not not related to a specific FreshDesk ticket.
-  This channel is public, so can be useful if we wish to invite conversation from non 2i2c team members.
+We use these channels for communicating with external stakeholders like Community Representatives:
 
-## Lifecycle of a support request
+- **[support@2i2c.org](mailto:support@2i2c.org)** is our point-of-contact for all support-related external communication.
+- **[The 2i2c FreshDesk account](https://2i2c.freshdesk.com/)** is where we track all support requests and _respond_ to emails sent to `support@2i2c.org`.
+- **{doc}`the "Get Support" page <docs:support>`** provides guidance that communities may follow to get support.
 
+### Internal communication
 
-Below is a summary of the major steps that make up the support process, from beginning to end of a ticket.
+We have a few channels for communicating around support requests:
 
-:::{note}
-We should try to respond to all support-related communications within one working day.
-:::
+- Our [FreshDesk account](https://2i2c.freshdesk.com/a/) allows for internal team communication via the {guilabel}`Add Note` button. This can be useful for sharing quick internal updates.
+- [Issues with the {guilabel}`support` label](https://github.com/2i2c-org/infrastructure/issues?q=is%3Aopen+label%3Asupport+sort%3Aupdated-desc) are where we track support requests related to {term}`Change Request`s and {term}`Guidance Request`s.
+
+## Support triage and resolution process
+
+Here is the process that we follow when triaging and resolving support requests.
+
+### Triage process
+
+The goal of the triage phase is to understand the Support Request, decide if it is related to an incident, and choose the appropriate resolution pathway.
+
+This process is carried out in an ongoing basis by the {term}`Support Steward`s.
+
+1. **Monitor our support channels**. We use FreshDesk for all support requests, and the Support Stewards should regularly keep an eye on this account for new requests.
+   When a new support requests comes in, move to the next step.
+2. **Read and understand**. Within 8 working hours, read the support request and try to understand what action would resolve it.
```

Comment:
I would suggest 24hs here.

Reply:
Ah... OK, this is the reading phase, not the response... then I would suggest 12hs (because of TZs).

Reply:
Yeah I guess maybe we should use something like "24 non-weekend hours" instead of "8 working hours". I think the general point we want to get across is "within 1 business day" (thus "8 working hours") but this is hard to convey when time-zones mean everybody has a different set of working hours.

Do you think "24 working hours" makes sense then, and maybe we add a note at the top of the document about how we define "working hours" so that others don't think that we actually mean "3 business days" when they read it?

Reply:
@choldgraf I definitely read this as 3 working days!

Reply:
> Do you think "24 working hours" makes sense then, and maybe we add a note at the top of the document about how we define "working hours"

Yep, I think we need to clarify the unit (whatever unit we are going to use) at the top so everyone knows what we are talking about 😉.

Reply:
I tried to add a clarification about this let me know what you think